app/helpers/application_helper.rb

```diff
@@ -5,6 +5,10 @@ def application_name
     'Argo'
   end
 
+  def search_params
+    params.except(:controller, :action).permit!
+  end
+
   def location_to_send_search_form
```

Comment:
move this to a helper since we use it in two places now (though actually it will go away for bulk update soon...)